web/routing.go

```diff
@@ -156,6 +157,7 @@ func SetupRoutes(router *echo.Echo) error {
 		middlewares.LoadSession,
 	}
 	router.GET("/", auth.Home, mws...)
+	oauth.Routes(router.Group("/oauth", mws...))
```

Comment:
Can we mount these endpoints on something more specific like `/oauth-for-konnectors`? I'd like to avoid confusion with the /auth endpoints for OAuth where cozy is the service provider.